### PR TITLE
OTT fixes

### PR DESCRIPTION
**Pull request by Diet-Cola**
Just need to confirm world blacklist is working but should fix existing issues

### CODE REVIEW

File: paper/src/main/java/com/programmerdan/minecraft/simpleadminhacks/configs/OneTimeTeleportConfig.java [hunk 1]
```diff
@@ -47,6 +50,15 @@ protected void wireup(ConfigurationSection config) {
 			}
 			materialBlacklist.add(material);
 		}
+		this.worldBlacklist = new ArrayList<>();
+		List<String> worlds = config.getStringList("home_world_blacklist");
+		for (String world : worlds) {
+			World realWorld = Bukkit.getWorld(world);
+			if (realWorld == null) {
+				continue;
+			}
```
Comment (Protonull) [inline paper/src/main/java/com/programmerdan/minecraft/simpleadminhacks/configs/OneTimeTeleportConfig.java:59]: I don't think there's any need to check whether the world exists on wireup. If it doesn't exist then it'll never match with a given location's world. You'd only really want this if you wanted to log a warning about an unknown world name in the config.

**Comment by Protonull** [comment]
This would seem to grant everyone a single use of OTT regardless of whether they're a new player or have been playing since SOTW. It also removes any time limit on the use of OTT, meaning that, in a year's time, they could possibly use this to get out of a sticky fight in a war.

**Comment by wingzero54** [comment]
Superseded by #70 , if we want the dimension blacklist this PR will need to be redone I believe?

**Comment by Protonull** [comment]
> Superseded by #70 , if we want the dimension blacklist this PR will need to be redone I believe?

Yeah, the dimension blacklist felt like an XY solution: that the goal was to fix pealed players from OTT'ing out of the nether, so the proposed solution here was to blacklist the nether. I figured it'd be better to check whether the player is pearled instead. I also added a configurable option to limit OTTs to within the same world. But yes, if you'd like to prevent OTTs to and from the nether regardless, this PR could serve as the addition to that feature, after having rebased, of course.

**Comment by RedDevel2** [comment]
Already fixed a while ago

**Pull request closed without merging**